script/polling-heartbeat-test.js

```diff
@@ -0,0 +1,74 @@
+"use strict";
+
+var assert, config, Plugin;
+
+assert = require("assert");
+config = require("./manual-testing-config");
+Plugin = require("../../../");
```

Comment:
I am presuming this is required by the plugin and the setup is the same as the MySql plugin. What should I do?